stellarphot/utils/magnitude_transforms.py

```diff
@@ -603,39 +605,42 @@ def fake_it(one_image):
         cat_colors.extend(nana)
         resids.extend(nana)
 
-    for file, one_image_inp in zip(
-        observed_mags_grouped.groups.keys, observed_mags_grouped.groups, strict=True
-    ):
-        one_image = one_image_inp[one_image_inp["passband"] == obs_filter]
-        our_coords = SkyCoord(one_image["ra"], one_image["dec"], unit="degree")
-        if cat is None or cat_coords is None:
+    one_coord = SkyCoord(
+        observed_mags_grouped["ra"][0], observed_mags_grouped["dec"][0], unit="degree"
+    )
+    if cat is None or cat_coords is None:
+        if cat_name == "apass_dr9":
             cat = apass_dr9(
-                our_coords[0], radius=1 * u.degree, clip_by_frame=False, padding=0
+                one_coord, radius=1 * u.degree, clip_by_frame=False, padding=0
```

Comment:
Not a complaint, but isn't 1 degree a pretty large search radius for APASS DR9 or is it really so few stars?

Reply:
our fov is about 0.5 degree, and the 1 degree search executes pretty quickly.